tests/run_all_tests.sh

```diff
@@ -1,5 +1,22 @@
 #!/bin/bash
 
+# Download TFDS metadata to flax/.tdfs/metadata directory. 
+# This allows the tests to specify the `data_dir` when using tfds.testing.mock_data().
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)"
```

Comment:
I think `dirname "$0"` works.

Reply:
Instead of setting the DIR env var, you can start the script with `cd $(dirname "$0")`

Reply:
Done.